colorpair.h

```diff
@@ -0,0 +1,54 @@
+#ifndef COLORPAIR_H_GUARD
+#define COLORPAIR_H_GUARD
+
+#define KNRM  "\x1B[0m"

```

Comment:
Would be hard to get that these defines are meant for formatting. Possible to give better names?